wordpress/README.md

```diff
@@ -0,0 +1,280 @@
+# Quick Start
+
+This repository includes the necessary files to package and deploy [Wordpress](https://www.wordpress.org/), a open source content management system, into a private cloud environment using [Gravity](http://gravitational.com/gravity/).  
+
+This manifest uses the Open Elastic Block Store ([OpenEBS](https://openebs.io/) built into Gravity 7.0.0+.
+
+# System Requirements
+
+
+* A x86_64 Linux machine or a VM for building a Cluster Image that is running one of the [supported Linux distributions](requirements.md#linux-distributions).
+* Docker version 17 or newer. Run `docker info` before continuing to make sure
+  you have Docker up and running. We recommend following instructions on [installing Docker CE from Docker.com](https://docs.docker.com/install/)
+* You must be a member of the `docker` group. Run `groups` command to make sure
+  `docker` group is listed. If not, you can add yourself to the "docker" group via `sudo usermod -aG docker $USER`
+* You must have `git` installed to clone the example application repo.
+* A _target cluster_ of Linux nodes. It can be just one machine but this example quickstart will use three. The nodes in a target cluster must have at least 2GB of RAM and 40GB of free disk space. They must **_not_** have Docker or any other container runtime installed on them.
+* You must have `sudo` privileges on all nodes.
+
+
+## Getting the Tools
+
+Start by [downloading Gravity](https://gravitational.com/gravity/download/) and
+unpacking the archive. You should see the following files:
+
+```
+$ ls -l
+-rwxr-xr-x 1 user user 108093824 Apr 22 11:43 gravity
+-rwxr-xr-x 1 user user       137 Apr 22 11:43 install.sh
+-rw-r--r-- 1 user user     11357 Apr 22 11:43 LICENSE
+-rw-r--r-- 1 user user      2880 Apr 22 11:43 README.md
+-rwxr-xr-x 1 user user  84764672 Apr 22 11:43 tele
+-rwxr-xr-x 1 user user  57863592 Apr 22 11:43 terraform-provider-gravity
+-rwxr-xr-x 1 user user  32488888 Apr 22 11:43 tsh
+-rw-r--r-- 1 user user         6 Apr 22 11:43 VERSION
+```
+
+Execute `install.sh` to copy `tele` and `tsh` binaries to
+`/usr/local/bin/`. Then you can type `tele version` to confirm that
+everything works:
+
+```
+$ tele version
+Edition:        open-source
+Version:        7.0.1
+Git Commit:     af393fcddf8c675f00cf322ec054125d5e239727
+Helm Version:   v2.15
+```
+Clone the sample Git repository which contains the Kubernetes resources for
+[Wordpress](https://www.wordpress.org/), which we are using in this tutorial as a sample application:
+
+```bsh
+$ git clone https://github.com/gravitational/quickstart.git
+$ cd quickstart
+```
+### Step 1: Containerizing
+
+Run the following to retrieve the required Wordpress containers:
+
+```bsh
+$ cd wordpress
+$ docker pull wordpress:4.8-apache && docker pull mysql:5.6
```

Comment:
This should not be necessary as `tele` will pull the images automatically.

Reply:
Removed this step